src/engine/sidechain/enginesidechain.cpp

```diff
@@ -75,11 +75,24 @@ void EngineSideChain::addSideChainWorker(SideChainWorker* pWorker) {
     m_workers.append(pWorker);
 }
 
-void EngineSideChain::writeSamples(const CSAMPLE* newBuffer, int buffer_size) {
+void EngineSideChain::receiveBuffer(AudioInput input,
```

Comment:
This is reimplementing a virtual method of AudioDestination so the name is required. There is a comment explaining this in the .h file. Would it be easier to read if this small wrapper function was inline in the .h file?

I don't think the additional check is really required. I followed the example of VinylControlProcessor::receiveBuffer which has an analogous check.